.github/workflows/edisonCI.yml

```diff
@@ -30,3 +30,6 @@ jobs:
         pip install flake8
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+    - name: Test with pytest
+      run: |
+        pytest --cov=edison tests/
```

Comment:
The "|" (pipe) purpose is for running multiple commands and because there's only one command executing you don't have to use it